src/client.js

```diff
@@ -1,5 +1,5 @@
 var _ = require("lodash");
-var package = require("../package.json");
+var packageJson = require("../package.json");
```

Comment:
https://github.com/thelounge/lounge/pull/399 did something similar with server code, but went with `pkg`. I have no preference over which one, but it would be nice to use the same one :-)